shared/login/forms/index.native.js

```diff
@@ -1,26 +1,48 @@
 // @flow
-import React from 'react'
-import {Box, Button, Icon, Text} from '../../common-adapters'
+import React, {Component} from 'react'
+import {Box, Button, HOCTimers, Icon, Text} from '../../common-adapters'
 import {globalColors, globalStyles, globalMargins} from '../../styles'
 
 import type {Props} from '.'
+import type {TimerProps} from '../../common-adapters/hoc-timers'
 
-const Splash = (props: Props) => (
-  <Box style={{...stylesLoginForm, justifyContent: 'center'}}>
-    <Icon type="icon-keybase-logo-80" />
-    <Text style={stylesHeader} type="HeaderBig">Keybase</Text>
-    <Box style={globalStyles.flexBoxColumn}>
-      <Text style={{marginTop: globalMargins.large}} type="BodySmall">
-        Keybase not starting up?
-      </Text>
-      <Button
-        label="Let us know"
-        onClick={props.onFeedback}
-        style={{marginTop: globalMargins.small}}
-        type="Primary"
-      />
-    </Box>
-  </Box>
+const Splash = HOCTimers(
```

Comment:
just fyi you could use recompose and not have to build the class, you can use compose(withState, lifecycle) etc